### PR TITLE
Update map hover tooltip styles

### PR DESCRIPTION
**Pull request by chiaberry**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6253

To test:
https://deploy-preview-365--atd-moped-main.netlify.app/moped/projects/276?tab=summary

hover over street names to bring up tooltip

### CODE REVIEW

File: moped-editor/src/utils/stringToTitleCase.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default str => {
+  let lowerStr = str.toLowerCase();
+  // uppercase first character
+  lowerStr = lowerStr.charAt(0).toUpperCase() + lowerStr.slice(1);
+  // uppercase characters after spaces
+  return lowerStr.replace(/\W\S/g, t => t.toUpperCase());
+};
```
Comment (sergiogcx) [inline moped-editor/src/utils/stringToTitleCase.js:7]: Absolutely brilliant, but I wonder if this is a necessity or an overkill? CSS's `text-transform: capitalize;` could probably work just fine without the need of javascript:

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

**Comment by sergiogcx** [comment]
Thank you for fixing it! I love the new style and the box-shadow 👍 

I did notice the text style switches back to times-new-roman in the new project form map:

File: moped-editor/src/utils/stringToTitleCase.js (hunk 1 above)
Comment (chiaberry) [reply moped-editor/src/utils/stringToTitleCase.js:7]: good chance that I overengineered it, but the hovertext is coming back ALLCAPS, does capitalize still work with that? 

ah, I could just make lowercase and then use css to capitalize. 👍🏼

**Comment by chiaberry** [comment]
thank you for catching the font change! ill fix now

**Comment by amenity** [comment]
LSTM! Roboto in all the places... 🤖

**Review by amenity — APPROVED** [review]

**Comment by chiaberry** [comment]
I re-requested a review from @sergiogcx since I made the udpates and it reset @amenity's approval :(

**Review by sergiogcx — APPROVED** [review]
It looks very classy now! 💯 😎 🙏

**Comment by sergiogcx** [comment]
@chiaberry it should be good to merge imo, since @amenity already approved

**Review by amenity — APPROVED** [review]
@chiaberry - good to merge!

**Pull request merged**